tests/OSPSuite.SimModel.Tests/SimulationSpecs.cs

```diff
@@ -1366,4 +1366,26 @@ public void should_retrieve_solver_warnings()
       }
    }
 
+   public class when_exporting_model_to_cpp_code : concern_for_Simulation
+   {
+      protected override void OptionalTasksBeforeLoad()
+      {
+         sut.Options.KeepXMLNodeAsString = true;
+      }
+
+      [Observation]
+      public void should_export_to_cpp()
+      {
+         var exportFolder = CreateTempFolder();
+
+         LoadSimulation("CPPExportTest01");
+         sut.ExportToCode(exportFolder, CodeExportLanguage.Cpp, CodeExportMode.Formula, "CPPExportTest01_Formula");
+         File.Exists(Path.Combine(exportFolder, "CPPExportTest01_Formula.cpp")).ShouldBeTrue();
+
+         LoadSimulation("CPPExportTest01");
+         sut.ExportToCode(exportFolder, CodeExportLanguage.Cpp, CodeExportMode.Values, "CPPExportTest01_Values");
+         File.Exists(Path.Combine(exportFolder, "CPPExportTest01_Values.cpp")).ShouldBeTrue();
+      }
+
```

Comment:
Finally I was able to compile this test model both in _formula_ and in _value_ mode
https://ci.appveyor.com/project/open-systems-pharmacology-ci/compiledmodel-template/builds/39796781
Did not try to run it though